### PR TITLE
ResourceController service update

### PR DESCRIPTION
**Pull request by apoorva9s14**
Signed-off-by: apoorva9s14 <apoorvalaiti16@gmail.com>

## PR summary
For the resource controller service update as per this PR - https://github.ibm.com/cloud-api-docs/resource-controller/pull/244, resource-controller-v2.yaml is the updated yaml, I have regenerated the SDK files with openapi-sdkgen.
We are looking to address:

Looping - https://github.ibm.com/cloud-api-docs/resource-controller/issues/226
Credentials redaction

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ Yes] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ Yes] Docs have been added / updated (for bug fixes / features)

## Current vs new behavior  
The examples and the SDK code snippet in the apidocs would be updated for credentials redaction

## Does this PR introduce a breaking change?    
- [ ] Yes
- [. ] No

### CODE REVIEW

File: resourcecontrollerv2/resource_controller_v2.go [hunk 1]
```diff
@@ -2266,6 +2266,12 @@ func (options *CreateResourceKeyOptions) SetHeaders(param map[string]string) *Cr
 
 // Credentials : The credentials for a resource.
 type Credentials struct {
+	// If present, the user doesn't have the correct access to view the credentials and the details are redacted.  The
+	// string value identifies the level of access that's required to view the credential. For additional information, see
+	// [viewing a
+	// credential](https://cloud.ibm.com/docs/account?topic=account-service_credentials&interface=ui#viewing-credentials-ui).
+	Redacted *string `json:"REDACTED,omitempty"`
```
Comment (dpopp07) [inline resourcecontrollerv2/resource_controller_v2.go:2273]: So where is this coming from? Was it added by hand? I don't see it in the API definition

**Review by dpopp07 — COMMENTED** [review]
I think the changes look good but I left a question about the credentials redaction piece to review before approving

File: resourcecontrollerv2/resource_controller_v2.go (hunk 1 above)
Comment (apoorva9s14) [reply resourcecontrollerv2/resource_controller_v2.go:2273]: We have updated the api definition, the PR is merged in test, It needs to be merged in production - https://github.ibm.com/cloud-api-docs/resource-controller/pull/248

**Review by dpopp07 — APPROVED** [review]
Looks good to me 👍

**Pull request merged**